### PR TITLE
[17.0][MIG] product_variant_specific_description

### PR DESCRIPTION
**Pull request by David-Luis-Mora**

### CODE REVIEW

File: product_variant_specific_description/views/product_template_view.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+    <record id="product_template_only_form_view" model="ir.ui.view">
+        <field name="name">product.template.product.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='description']/.." position="attributes">
+                <attribute name="invisible">'product_variant_count', '>', 1</attribute>
```
Comment (MarioLM-23) [inline product_variant_specific_description/views/product_template_view.xml]:
```suggestion
                <attribute name="invisible">product_variant_count &gt; 1</attribute>
```

File: product_variant_specific_description/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product Variant Specific Description",
+    "version": "17.0.1.0.1",
```
Comment (MarioLM-23) [inline product_variant_specific_description/__manifest__.py:6]:
```suggestion
    "version": "17.0.1.0.0",
```

File: product_variant_specific_description/tests/test_product_variant_description.py
Comment (MarioLM-23) [inline product_variant_specific_description/tests/test_product_variant_description.py:1]: SetUpClass is a better option.

**Review by MarioLM-23 — CHANGES_REQUESTED** [review]
LGTM but there are some errors in product template view, manifest and an improvement in the tests.

**Review by gurneyalex — CHANGES_REQUESTED** [review]
please remove the pandoc-3.1.13-1-amd64.deb file

**Pull request closed without merging**